apps/demo-game/src/pages/admin/games/[id].tsx

```diff
@@ -441,6 +454,30 @@ function ManageGame() {
           ))}
         </div>
       </div>
+      <div className="mt-4">
+        <Formik
+          initialValues={{ countdownSeconds: 300 }}
+          onSubmit={(values) =>
+            addCountdown({
+              variables: {
+                gameId: Number(router.query.id),
+                seconds: Number(values.countdownSeconds),
+              },
+            })
+          }
+        >
+          {(countdownForm) => (
+            <div className="flex flex-row gap-2">
+              <FormikNumberField name="countdownSeconds" precision={0} />
+              <Button onClick={() => countdownForm.handleSubmit()}>
```

Comment:
normally it is preferable to add type="submit" to the main button of a form, which also triggers the submit function of formik when the button is pressed and additionally makes it clear in terms of accessibility that it is the actual submit button. so we don't actually have to call the handle submit function (and here it would also mean that we don't even need function as a child, can just pass the div as a child